ExampleApp/ExampleViewController.swift

```diff
@@ -126,6 +126,19 @@ class ExampleViewController: UIViewController, UITableViewDataSource, UITableVie
                 ]
             ),
             Section(
+                label: "Debugging",
+                actions: [
+                    Action(
+                        label: "deleteLinkedBrokers",
+                        action: deleteLinkedBrokers
+                    ),
+                    Action(
+                        label: "test",
+                        action: test
+                    )
+                ]
+            ),
+            Section(
```

Comment:
Moving this up - our UI tests fail if it's off-screen and swiping up/down is slow.